src/containers/shared/utils.js

```diff
@@ -151,7 +151,12 @@ export const getLocalizedCurrencySymbol = (lang = 'en-US', currency = 'USD') =>
   return formatted.split('1')[0].trim();
 };
 
+// displaying 4 digits will give user a sense of how small the number is.
+// less number of digits might not have siginificant information since tokens have really small value.
```

Comment:
This comment isn't needed anymore, now that we changed `variableDigits` from a magic number.